tests/test_registration_embargoes.py

```diff
@@ -8,7 +8,7 @@
 from tests.base import fake, OsfTestCase
 from tests.factories import (
     AuthUserFactory, EmbargoFactory, NodeFactory, ProjectFactory,
-    RegistrationFactory, UserFactory, UnconfirmedUserFactory
+    RegistrationFactory, UserFactory, UnconfirmedUserFactory, DraftRegistrationFactory
```

Comment:
I think it's important that when we have opinions about style we make it clear that we are asserting an opinion.